src/meson.build

```diff
@@ -238,4 +246,7 @@ src = meson.current_source_dir()
 
 # Submodules
 subdir('sage')
-subdir('doc')
```

Comment:
I think it's a tad cleaner if this check is in `doc/meson.build` similar to https://github.com/vbraun/sage/commit/9573c67a62fe4414d1bed1778a8e646d051c892c